textproc/cgrc/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           qt6 1.0
+
+fetch.type          git
+github.setup        carlonluca cgrc v0.3.2
+version             0.3.2
+revision            1
+license             GPL-3
+categories          textproc
+platforms           darwin
+maintainers         {@carlonluca gmail.com:carlon.luca} openmaintainer
+homepage            https://github.com/carlonluca/cgrc
```

Comment:
`homepage` is implicitly generated by `github.setup` - you can remove this line.